charts/cluster-scanner/values.yaml

```
@@ -0,0 +1,124 @@
+#basic Default values for cluster-scanner.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+global:
+  sysdigHost: ""
+  sysdigAccessKey: ""
+  clusterName: ""
+  sysdigSkipVerifyCertificate: false
+
+  scannerMode: "local" # or "multi"
+  loggingLevel: "INFO"
+
+eveEnabled: false
+eveIntegrationEnabled: false
+rootNamespace: "kube-system"
+
+multicluster:
```

We are using camelCAse for all the other config, I suggest to keep consistency.
```suggestion
multiCluster:
```

---

This seels to be specific for rsi, can we move it as config in that section?

---

same reasons as per https://github.com/sysdiglabs/charts/pull/1056#discussion_r1173869100

---

Marking this as resolved since we moved this configuration under the `runtimeStatusIntegrator` itself.